docs/reference/tctl/batch/list.md

```diff
@@ -24,3 +24,8 @@ Alias: `--ps`
 ```
 tctl batch list --pagesize <value>
 ```
+
+## Related
+
+- [`tctl batch`](../batch)
+- [tctl reference](/docs/reference/tctl)
```

Comment:
I am thinking that we can get creative with a sidebar to solve this problem - without compromising on the good stuff.

Reply:
This is most excellent. Thanks for pioneering!